config/versions/versions.go

```diff
@@ -59,7 +61,7 @@ type manager struct {
 var Manager = &manager{}
 
 // Deploy upgrades or downgrades the config between versions
-// version param -1 defaults to the latest version
+// version param LatestVersion defaults to the latest version
```

Comment:
revert?

Reply:
No, this was deliberate. I think I can word it better though.

Fixed https://github.com/gbjk/gocryptotrader/commit/cc91852e0